### PR TITLE
update pytorch and cuda versions

### PR DESCRIPTION
**Pull request by jbkyang-nvi**

### CODE REVIEW

File: qa/L0_backend_python/bls/test.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ rm -fr *.log ./models *.txt
 # FIXME: [DLIS-5970] Until Windows supports GPU tensors, only test CPU
 if [[ ${TEST_WINDOWS} == 0 ]]; then
     pip3 uninstall -y torch
-    pip3 install torch==1.13.0+cu117 -f https://download.pytorch.org/whl/torch_stable.html
+    pip3 install torch==2.2.0+cu121 -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (mc-nv) [inline qa/L0_backend_python/bls/test.sh:39]: It looks windows related changes may be require classifier: 
```
$ curl https://download.pytorch.org/whl/torch_stable.html | cut -d '"' -f 2 | grep /torch-2 | grep "win" | grep 121
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  626k  100  626k    0     0  1438k      0 --:--:-- --:--:-- --:--:-- 1437k
cu121/torch-2.1.0%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.0%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.1%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.1.2%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp312-cp312-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.2.0%2Bcu121-cp39-cp39-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp310-cp310-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp311-cp311-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp312-cp312-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp38-cp38-win_amd64.whl
cu121/torch-2.2.1%2Bcu121-cp39-cp39-win_amd64.whl
```

File: qa/L0_backend_python/examples/test.sh [hunk 2]
```diff
@@ -37,9 +37,9 @@ rm -fr *.log python_backend/
 # Install torch
 pip3 uninstall -y torch
 if [[ "$TEST_JETSON" == "0" ]] || [[ ${TEST_WINDOWS} == 0 ]]; then
-    pip3 install torch==2.0.0+cu117 -f https://download.pytorch.org/whl/torch_stable.html torchvision==0.15.0+cu117
+    pip3 install torch==2.2.0+cu121 -f https://download.pytorch.org/whl/torch_stable.html torchvision==0.16.0+cu121
```
Comment (mc-nv) [inline qa/L0_backend_python/examples/test.sh:40]: This output of `pip3 list` from `nvcr.io/nvidia/pytorch:24.01-py3` image  shows that `torchvision` version needs to be higher 
```
pytorch-quantization      2.1.2
torch                     2.2.0a0+81ea7a4
torch-tensorrt            2.2.0a0
torchdata                 0.7.0a0
torchtext                 0.17.0a0
torchvision               0.17.0a0
```

File: qa/L0_backend_python/bls/test.sh (hunk 1 above)
Comment (jbkyang-nvi) [reply qa/L0_backend_python/bls/test.sh:39]: This one is not for Windows I believe

**Comment by mc-nv** [comment]
@jbkyang-nvi , please change the branch or close the PR

**Pull request closed without merging**